framework/src/outputs/DOFMapOutput.C

```diff
@@ -53,14 +53,14 @@ DOFMapOutput::DOFMapOutput(const InputParameters & parameters) :
     _mesh(_problem_ptr->mesh())
 {
   // Set output coloring
-  if (Moose::_color_console)
+  if (Moose::colorConsole())
   {
     char * term_env = getenv("TERM");
     if (term_env)
     {
       std::string term(term_env);
       if (term != "xterm-256color" && term != "xterm")
-        Moose::_color_console = false;
+        Moose::setColorConsole(false);
```

Comment:
Uhm, yeah... I wrote the DOFMap output object, but I have no idea why I thought it'd be a good idea to have an individual output object mess with the global output coloring flag. This was probably copy pasted from `Console`. This output object should just write to files anyways! Just remove this entire if block.